common/djangoapps/student/migrations/0049_auto__add_languageproficiency__add_unique_languageproficiency_code_use.py

```diff
@@ -173,7 +173,7 @@ def backwards(self, orm):
             'courseware': ('django.db.models.fields.CharField', [], {'default': "'course.xml'", 'max_length': '255', 'blank': 'True'}),
             'gender': ('django.db.models.fields.CharField', [], {'db_index': 'True', 'max_length': '6', 'null': 'True', 'blank': 'True'}),
             'goals': ('django.db.models.fields.TextField', [], {'null': 'True', 'blank': 'True'}),
-            'has_profile_image': ('django.db.models.fields.BooleanField', [], {'default': 'False'}),
+            'profile_image_version': ('django.db.models.fields.CharField', [], {'max_length': '16', 'null': 'True'}),
```

Comment:
when we introduced this it was meant for use as a stamp/fingerprint to make the urls change whenever the image changed, but it was not intended to be used as file metadata.  AFAIK we don't have any specific requirement to track the user's last image upload time.  Also, we already have the modification time of the physical files which is accessible via profile image storage (maybe also by HEAD-ing the file URLs).

I'm personally not in favor of overloading what was originally designed as an opaque version stamp, but if everyone else feels it's better, I'll rename the field and change the type in the database to a datetime stamp (which i think makes more sense than a server-generated int/float).